variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/writers/VariantMongoWriter.java

```diff
@@ -146,13 +174,13 @@ private DBObject generateUpdate(IVariant variant) {
             addToSet.put(IDS_FIELD, new BasicDBObject("$each", variant.getIds()));
         }
 
-        BasicDBObject update = new BasicDBObject();
+        if (variant.getDbsnpIds() != null && !variant.getDbsnpIds().isEmpty()) {
```

Comment:
I guess this is more of a defensive code, right? because getDbsnpIds doesn't look like it would return null in any case.

Reply:
hmm, yes, I'll take it out.